### PR TITLE
Fixed spacing between edit and delete buttons

### PR DESCRIPTION
**Pull request by noisecapella**
This looks like a whitespace change but it translates differently from JSX.

### CODE REVIEW

File: ui/static/ui/js/manage_taxonomies.jsx [hunk 1]
```diff
@@ -141,8 +141,7 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
               <span className="utility-features">
                 <a href="#">
                   <i className="fa fa-pencil"></i>
-                </a>
-                <a data-toggle="modal" data-target="#confirm-delete"
+                </a> <a data-toggle="modal" data-target="#confirm-delete"
```
Comment (giocalitri) [inline ui/static/ui/js/manage_taxonomies.jsx:144]: what is the difference? The original spacing should be collapsed to 1 space, shouldn't it?

Comment (noisecapella) [inline ui/static/ui/js/manage_taxonomies.jsx:144]: No actually, they're just put right next to each other. It's a quirk of JSX

Comment (noisecapella) [inline ui/static/ui/js/manage_taxonomies.jsx:144]: You can confirm this by going to https://facebook.github.io/react/jsx-compiler.html and trying out:

```
<div>
<a /> <a />
</div>
```

and comparing the output to

```
<div>
<a /> 
<a />
</div>
```

**Comment by giocalitri** [comment]
this is good to be merged

**Pull request merged**